test/sagemaker_tests/pytorch/inference/integration/sagemaker/test_mnist.py

```diff
@@ -118,3 +124,50 @@ def _test_mnist_distributed(
         output = predictor.predict(data)
 
         assert output.shape == (batch_size, 10)
+
+        #  Check for Cloudwatch logs
+        if verify_logs:
+            _check_for_cloudwatch_logs(endpoint_name)
+
+def _check_for_cloudwatch_logs(endpoint_name):
+    client=boto3.client('logs')
+    identifyLogStream = client.describe_log_streams(
+        logGroupName='/aws/sagemaker/Endpoints/'+endpoint_name,    
+        orderBy='LastEventTime',
+        descending=True,
+        limit=5
+    )
+
+    indexError=False;
+    try:
+        logStreamName=identifyLogStream['logStreams'][0]['logStreamName']        
+    except IndexError:
+        indexError=True    
+
+    if(indexError):
+        raise Exception('Exception: Trying to lookup log streams from the given log group failed')
+
+    log_events_response = client.get_log_events(
+        logGroupName='/aws/sagemaker/Endpoints/'+endpoint_name,
+        logStreamName=logStreamName,
+        limit=50,
+        startFromHead=True
+    )
+
+    #print(log_events_response['events'])
+    recordsAvailable=bool(log_events_response['events'])
+    print('Are log events avaiable? '+str(recordsAvailable))
+   
+    if not recordsAvailable:        
+        raise Exception('Exception: No cloudwatch events getting logged for the group /aws/sagemaker/Endpoints/'+endpoint_name)
+    else:    
+        print('INFO: Most recently logged events were found for the given log group & log stream... Now verifying that TorchServe endpoint is logging on cloudwatch')
+        check_for_torchserve_response = client.filter_log_events(
+            logGroupName='/aws/sagemaker/Endpoints/'+endpoint_name,
+            logStreamNames=[logStreamName], 
+            filterPattern='Torch worker started.',
+            limit=10,
+            interleaved=False
+        )
+        assert bool(check_for_torchserve_response['events'])
+        print('Success!')
```

Comment:
you can likely remove most of the print debug statements unless you think they would be useful. If you think useful, we should use python logging and set to "debug" level

Reply:
Done.. Removed all debug/print and replaced them with logger implementation